db/migrate/20180118002015_create_datasets_v2.rb

```diff
@@ -0,0 +1,39 @@
+class CreateDatasetsV2 < ActiveRecord::Migration
+  def change
+    create_table :datasets do |t|
+
+      t.string :name
+      t.text :description
+      t.integer :creator_id
+      t.timestamps null: false
+
+    end
+    create_table :dataset_items do |t|
+
+      t.integer :dataset_id
+      t.integer :audio_recording_id
+      t.decimal :start_time_seconds, :null => false
+      t.decimal :end_time_seconds, :null => false
+      t.decimal :order
+      t.datetime :created_at
+
+    end
+    create_table :progress_events do |t|
+
+      t.integer :user_id
+      t.integer :dataset_item_id
+      t.string :activity
+      t.datetime :created_at
```

Comment:
I'd recommend adding :creator_id to be consistent with other tables. It will likely always be the same user as :user_id. However, if at some point a user (e.g. admin, dataset owner) is able to manually change progress, then it will not be the same user.

Reply:
yeah I had this argument in my head as well.

I thought that this table was analogous to permissions... we are recording an event that happens to a user, not who created the event. I thought that distinction was worth the deviation?